components/ui/terminal/floating-terminal-button.tsx

```
@@ -0,0 +1,44 @@
+      onClick={restoreWindow}
+      className={cn(
+        "fixed bottom-4 right-4 z-[100] p-3 rounded-full",
+        "bg-blue-600 hover:bg-blue-700 text-white",
+        "shadow-lg transition-opacity duration-300 ease-in-out",
+        "focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2",
+        isVisible ? "opacity-100 pointer-events-auto" : "opacity-0 pointer-events-none"
+      )}
+      aria-label="Restore Terminal"
+      title="Restore Terminal"
+      disabled={!isVisible} // Disable button when not visible to prevent accidental interaction
+    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)